backends/advanced/src/advanced_omi_backend/routers/modules/annotation_routes.py

```
@@ -1,625 +1,115 @@
+        segment_index=new_annotation.segment_index,
+        original_text=new_annotation.original_text,
+        corrected_text=new_annotation.corrected_text,
+        status=new_annotation.status,
+        created_at=new_annotation.created_at
+    )
+
+@router.get("/{conversation_id}", response_model=List[AnnotationResponse])
+async def get_annotations(
     conversation_id: str,
-    current_user: User = Depends(current_active_user),
+    current_user: User = Depends(current_active_user)
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)